### PR TITLE
Pop up window

### PR DESCRIPTION
**Pull request by ZabihullahNooriWardak**
in this branch we worked on following features
● pop up window added for mobile version.
● pop up window added for desktop version.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -25,3 +25,206 @@ const LiInMenu = document.getElementsByClassName('menue');
...
+       consectetur adipisicing elit. Quos illum ex ducimus quaerat
+        inventore officia blanditiis impedit suscipit,
+         unde nulla, porro magnam
+          incidunt corporis quod 
+         repellat mollitia eum nesciunt! Sunt!</p>
+   <div class='popUpFooter'>
+   <button>See live</button>
+   <button>See source </button>
+   </div>
+  `;
```
Comment (divinecharlotte) [inline script.js:217]: - [ ] kindly consider sticking to the given design as much as possible  (e.g., font, colors, images, text, margins) here am emphasizing margins.to make your website an. exact duplicate of the provided Figma either on mobile popup or desktop popup check the images below to see what needs to be fixed
- your mobile popup design 👇 
Screen Shot 2023-06-29 at 14 03 49

- your desktop pop-up design 👇 
Screen Shot 2023-06-29 at 14 00 46

- given mobile popup design 👇 
Screen Shot 2023-06-29 at 14 18 29

- given desktop pop-up design 👇 
Screen Shot 2023-06-29 at 14 19 55

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Team  , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Descriptive PR👍 
- Descriptive readme 👍 

 




 

## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 2]
```diff
@@ -25,3 +25,206 @@ const LiInMenu = document.getElementsByClassName('menue');
...
+   <i class='bx bx-x'></i>
+   </div>
+   <ul>
+   <li>HTML</li>
+   <li>Bootstrap</li>
+   <li>Ruby on Rails</li>
+   </ul>
+${dynamicImage}
+   <p>             Lorem ipsum dolor sit amet
+    consectetur adipisicing elit.
```
Comment (V-Blaze) [inline script.js:187]: - [ ] You have done great so far :+1: :+1: But there are still some improvements you need to make to the UI.
- - As the previous reviewer pointed out, the pop-up for the desktop screens doesn't match the Figma design well. Looking at the Figma, you can see that the image covers about 60% of the pop-up while the text takes about 40%. This seems to be the reverse in your project. I'll suggest you update this. :smile: :wave: 
- - SCREENSHOT FIGMA:
Screenshot from 2023-06-29 16-29-27

- - YOUR DESIGN:
Selection_072

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @ZabihullahNooriWardak , 

# Almost there 🟡 

1094a3e620ebea4cd6cf2d0aceaa8f52


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing some changes from the previous reviewers, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
-  Linters setup correctly✔
- Correct GitHub flow💯 🥇 ✔

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @ZabihullahNooriWardak   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :fire: 
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**